python/tank/bootstrap/resolver.py

```diff
@@ -255,31 +264,48 @@ def resolve_shotgun_configuration(
             # resolve primary and user config
             primary_config = None
             user_config = None
+            secondary_primary_config = None
+            secondary_user_config = None
```

Comment:
Might be worth tweaking the comment above these lines to reflect these additions. Also, maybe there's a better prefix than `secondary_` ? `secondary_primary_` is confusing.